scrapers/freshmaker.py

```diff
@@ -70,8 +70,30 @@ def query_api_and_update_neo4j(self):
 
                 for build_dict in fm_event['builds']:
                     # To handle a faulty container build in Freshmaker
-                    if not build_dict['build_id'] or int(build_dict['build_id']) < 0:
```

Comment:
Why was this removed? There was a Freshmaker bug at one point that caused the IDs to be negative and I think those are still presented in the API. If that's the case, then the scraper still needs to be able to ignore them.

Reply:
I'll put it back. I thought the negative build id meant failed build. The comment says "faulty container build in Freshmaker", is it related only to the first condition of the if?